docs/reference/experimental/async/activity.md

```diff
@@ -0,0 +1,20 @@
+# Activity
```

Comment:
This is the warning message I originally had when going to this page (https://python-docs.synapse.org/reference/oop/models/), should we add this back-in somewhere?

> Contained within this file are experimental interfaces for working with the Synapse Python
Client. Unless otherwise noted these interfaces are subject to change at any time. Use
at your own risk.